Makefile - make install-dev not installing d3-scale

Steps to reproduce it:
- Make sure node_modules directory does not exist
- Execute:

`# make install-dev`

Actual Behaviour:
Build process fails:

> Users/anunez9/Documents/Code/viri/ciao-webui/node_modules/d3-scale/index.js:1
> export {
> ^
> ParseError: 'import' and 'export' may appear only with 'sourceType: module'
> 
> npm ERR! Darwin 15.6.0
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build-group"
> npm ERR! node v6.2.0
> npm ERR! npm  v3.8.9
> npm ERR! code ELIFECYCLE
> npm ERR! ciao-webui@1.0.0 build-group: `browserify build/pages/tenant_group.js -o build/javascripts/bundle_tenant_group.js`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the ciao-webui@1.0.0 build-group script 'browserify build/pages/tenant_group.js -o build/javascripts/bundle_tenant_group.js'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the ciao-webui package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     browserify build/pages/tenant_group.js -o build/javascripts/bundle_tenant_group.js
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs ciao-webui
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls ciao-webui
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /Users/anunez9/Documents/Code/viri/ciao-webui/npm-debug.log

Expected Behaviour:
D3-scale should be successfully installed during make install-dev

Workaround:
Prior executing make install, install d3-scale:

```
# npm install d3-scale
# make install-dev
```

--- a/Makefile
+++ b/Makefile
@@ -35,6 +35,7 @@ install:
 	sed -i.bak "s~$(subst $\",,$(match))~$(subst $\",,$(match))\\$(bootstrap)\\$(subst $\",,$(framework))\\$(subst $\",,$(chosen))\\$(subst $\",,$(d3))\\ $(subst $\",,$(jquery))\\$(subst $\",,$(jqueryui))\\$(subst $\",,$(datamanager))\\$(subst $\",,$(loadDatamanager))\\ $(subst $\",,$(validations))~" views/$(subst $\",,$(file))
 	-mkdir -p $(DESTDIR)/ciao-webui/
 	-mkdir -p $(DESTDIR)/share/ciao-webui
+	-mkdir -p $(NPM_LINK)
 	-cp config/ciao_config.json $(DESTDIR)/share/ciao-webui/
 	-cp -rf * $(DESTDIR)/ciao-webui/
 	sed "s/\".\" #executable/\"$(subst $\/,\\/,$(DESTDIR))\"/" ./ciao-webui.sh > $(NPM_LINK)/ciao-webui
@@ -50,6 +51,7 @@ install-dev:
 	-mkdir node_modules
 	-cp -r vendor/* node_modules/
 	-npm install
+	-npm install d3-scale
 	# installing Babel and Browserify globally
 	-npm install --global babel-cli
 	-npm install --global browserify

